test/topology_custom/suite.yaml

```diff
@@ -26,6 +26,7 @@ skip_in_debug:
   - test_node_shutdown_waits_for_pending_requests
   - test_cdc_generation_clearing
   - test_cdc_generation_publishing
+  - test_random_failures
```

Comment:
should be in `run_in_debug`, not `skip_in_debug`

Reply:
fixed